spark-indexer/src/main/java/org/uniprot/store/spark/indexer/chebi/GraphSendVerticesMessageMapper.java

```diff
@@ -0,0 +1,39 @@
+package org.uniprot.store.spark.indexer.chebi;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.spark.graphx.EdgeTriplet;
+import org.uniprot.core.cv.chebi.ChebiEntry;
+import org.uniprot.core.cv.chebi.impl.ChebiEntryBuilder;
+
+import scala.Function1;
+import scala.Tuple2;
+import scala.collection.Iterator;
+import scala.collection.JavaConverters;
+
```

Comment:
brief javadoc description because it's not clear for a newcomer to know what this class does or how it fits in